pkg/retrieval/retrieval.proto

```diff
@@ -0,0 +1,15 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
```

Comment:
Would you move this file to new package `pkg/retrieval/pb`, similarly as it is now in pingpong package? This has been a recent change, to avoid cluttering the api of the package.